lms/envs/common.py

```diff
@@ -903,7 +854,7 @@
     # .. toggle_default: False
     # .. toggle_description: A "work-around" feature toggle meant to help in cases where some file uploads are not
     #   discoverable.  If enabled, will iterate through all possible file key suffixes up to the max for displaying
-    #  file metadata in staff assessments.
+    #   file metadata in staff assessments.
```

Comment:
**Note:** Dropping the ability to add comments in-between toggle annotations with a group would enable linting to have caught this issue, because there were more toggles in the group. The same indent issue above wouldn't have been caught because it happened to also be the last annotation. The line with the bad-indent in both cases would have been lost until we improve this issue.

This comment is to help with future planning.